Core/Src/analyzer.c

```diff
@@ -115,14 +115,16 @@ Timer ocvTimer;
 
 bool is_first_reading_ = true;
 
-void push(AccumulatorData_t* data)
+void push(AccumulatorData_t* data, sht30_t* sht30data)
```

Comment:
just gonna wanna update the acual call of this function in main.c to use this new param as well